Lib/test/test_traceback.py

```diff
@@ -4363,23 +4364,24 @@ def test_colorized_detection_checks_for_environment_variables(self):
             with unittest.mock.patch("os.isatty") as isatty_mock:
                 isatty_mock.return_value = True
                 with unittest.mock.patch("os.environ", {'TERM': 'dumb'}):
-                    self.assertEqual(traceback._can_colorize(), False)
+                    self.assertEqual(_colorize.can_colorize(), False)
```

Comment:
Should these tests be moved to a new `test__colorize.py` file?

Reply:
Sure, done, with some light refactoring.